cuqi/implicitprior/_regularizedGaussian.py

```diff
@@ -206,16 +256,18 @@ def sqrtcov(self):
     def sqrtcov(self, value):
         self.gaussian.sqrtcov = value     
     
-    def get_conditioning_variables(self):
-        return self.gaussian.get_conditioning_variables()
-    
     def get_mutable_variables(self):
-        return self.gaussian.get_mutable_variables()
+        add = []
```

Comment:
Cool to have strength as a mutable variable! I suggest this a bit refactoring
```python
def get_mutable_variables(self):
    mutable_vars = self.gaussian.get_mutable_variables()
    
    if self.preset in self.regularization_options():
        mutable_vars += ["strength"]
    
    return mutable_vars
```

Reply:
Thanks for the suggestion.